src/ChannelServer/Network/ChannelPacketHandler.cs

```diff
@@ -927,12 +930,12 @@ public void CZ_SKILL_TARGET(ChannelConnection conn, Packet packet)
 			var targetHandle = packet.GetInt();
 
 			var character = conn.SelectedCharacter;
+			var skill = character.Skills.Get(skillId);
 
-			if (character.CanCast(skillId))
+			if (skill != null)
```

Comment:
Was this change intended? Now you aren't checking and reducing the SP anymore, are you?

Reply:
It wasn't an intended change when I removed CanCast, readded it within the SkillHandlers.